modules/etcd-encryption-types.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * security/encrypting-etcd.adoc
+// * post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: CONCEPT
+[id="etcd-encryption-types_{context}"]
+= Supported encryption types
+
+The following encryption types are supported for encrypting etcd data in {product-title}:
+
+AES-CBC:: Uses AES-CBC with PKCS#7 padding and a 32 byte key to perform the encryption. The encryption keys are rotated on a weekly basis.
```

Comment:
In the spirit of minimalism, you can replace "on a weekly basis" here and on line 14 with "weekly". Your current wording is perfectly fine, though!